src/generate_plot_data.py

```diff
@@ -61,6 +61,17 @@ def __init__(self):
 
     for i in range(args.num_boots):
         y_true_resampled, y_pred_resampled = resample(y_true, y_pred)
+        
```

Comment:
I agree we need to look at this issue. However, it doesn't look like this leads anywhere as of yet. `car` doesn't get re-used as far as I can tell. Is this just for debugging purposes? If we need to address the issue, I would delete this block and add it to a separate issue instead.

Reply:
Yeah, I'm sorry about the car variable, that was just for debugging. I got rid of it, but left the remainder of my changed code because that caught any of the resampled stuff that didn't work. This is the most elegant way I could think of doing it, but there might be something better.

Reply:
Why don't we leave it out for now and then submit an issue. I have a few ideas on how we can implement a solution.